### PR TITLE
[CI] Fix typo

### PR DESCRIPTION
**Pull request by mandel-macaque**
Lets make my 1000 commits a typo fix!

### CODE REVIEW

File: tools/devops/automation/build-pipeline.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ parameters:
 #
 # 1. We can create a template that expands.
 # 2. We can use the each keyword, which allow a loop
-# 3. yaml is a super set of json, therefore, it was jsond dictionaties
+# 3. yaml is a super set of json, therefore, it was json dictionaties
```
Comment (spouliot) [inline tools/devops/automation/build-pipeline.yml:37]: why not both ? `dictionaties`

**Review by spouliot — CHANGES_REQUESTED** [review]

File: tools/devops/automation/build-pipeline.yml (hunk 1 above)
Comment (mandel-macaque) [reply tools/devops/automation/build-pipeline.yml:37]: Cause i did not see that!

Comment (mandel-macaque) [inline tools/devops/automation/build-pipeline.yml:37]:
```suggestion
# 3. yaml is a super set of json, therefore, it was json dictionaries
```

**Review by spouliot — APPROVED** [review]

**Comment by spouliot** [comment]
rename description to `typos` before merging :-)

**Pull request merged**